src/content/docs/configuration/providers.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Providers
+description: Understand Providers and configure new Targets.
+sidebar:
+  label: Providers
+---
+
+Daytona abstracts the management and deployment of Workspaces into 2 related concepts:
+
+1.  __Provider__  
+    A platform capable of deploying Workspaces on.
+    Examples of Providers may include:
+    *   Container management platforms such as Docker or Podman;
+    *   Cloud hosting providers such as AWS or DigitalOcean.
+2.  __Target__  
+    A set of configuration that governs how Daytona manages and deploys Workspaces.
+    Each Target is tied to an individual Provider.
+    Daytona allows you to configure and switch between multiple Targets at Workspace creation time.
+
+Daytona's architecture decentralizes the role of a Provider into a separate service.
+The [Daytona Server](/reference/server) communicates with Providers to execute operations relating to Workspace creation and lifecycle management.
+
+This makes it possible to create new Providers to govern Workspace management on any platform or hosting providers.
+
+## Managing Targets
+
+### Set a Target
+You can set a new Target for Daytona to use when managing Workspaces.
+
+__Prerequisites__
+*   A working installation of Daytona.
+*   At least one Provider installed.
+
+__Procedure__
+1.  Execute the following command:
+    ```shell
+    daytona target set
+    ```
+2.  Select the appropriate Provider for the environment you want to deploy to.
+3.  Select `New Target`.
+4.  Enter a name for your Target.
+5.  Enter the appropriate configuration options when prompted.
+6.  Click `Enter` to add the Target.
+
+### Remove a Target
+You can remove an existing target.
+Once a Target is removed, you will not be able to manage or deploy Workspaces without adding another Target with the same configuration.
+
+__Prerequisites__
+*   A working installation of Daytona.
+*   At least 1 configured Target.
+
+__Procedure__
+1.  Execute the following command to enter the interactive terminal UI:
+    ```shell
+    daytona target set
+    ```
+2.  Select the target to remove.
+
+## Using the Docker Provider
+After installation, Daytona installs a Provider for Docker (`daytona-provider-docker`) by default.
+A default Target using the Docker Provider is also added, allowing the creation of Workspaces on your local machine.
+
+### Add a Docker Target
```

Comment:
I'd just call this `Targets`

Reply:
I rearranged the hierarchy a bit in [92c92a0](https://github.com/daytonaio/docs/pull/24/commits/92c92a08286b290cdd0c774c7ac01248129d7f1a). Let me know if that looks good.